console/src/assets/i18n/en.json

```diff
@@ -42,6 +42,41 @@
       "ADD": "Hold and drag a tile to add"
     }
   },
+  "ONBOARDING": {
```

Comment:
translation missing: 
![image](https://user-images.githubusercontent.com/27845747/221162877-02258d32-f860-4414-a8b7-a9d6ae3aeaed.png)

Reply:
this card was just to test UI. Removed it.